crates/cli_bin/tests/snapshots/plumbing__checks_invalid_patterns.snap

```diff
@@ -1,10 +1,9 @@
 ---
-source: apps/marzano/cli/tests/plumbing.rs
+source: crates/cli_bin/tests/plumbing.rs
 expression: "String::from_utf8(result.stderr)?"
 ---
 Error: Failed to compile pattern `console.lo: engine_id: Some("marzano(0.1)")
-file_name: Some("PlaygroundPattern")
+file_name: None
```

Comment:
@arendjr @ilevyor This should *NOT* have been merged. Breaking this snapshot test causes lots of downstream problems.